src/open_inwoner/conf/parts/case_status_notification.html

```diff
@@ -79,16 +79,16 @@ <h2>{{ type_description }}</h2>
     <tr>
         <td class="td--responsive td-mail td__padding-right td__padding-bottom td__padding-left" colspan="3"><p><a href="https://example.com">Ga naar onze contactpagina</a></p>
         </td>
-        <td class="td--responsive td-mail td__padding-right td__padding-bottom td__padding-left" colspan="3"><p><a href="tel:+14050" data-rel="external">Bel 14050</a></p></td>
+        <td class="td--responsive td-mail td__padding-right td__padding-bottom td__padding-left" colspan="3"><p><a href="tel:+12345" data-rel="external">Bel 12345</a></p></td>
     </tr>
     <tr class="tr-mail border-top">
         <td class="td-mail border-top td__padding-zero" colspan="6">
             <!-- <hr> styling --> &nbsp;
         </td>
     </tr>
     <tr>
-        <td class="td-mail td__padding-all" colspan="6">
-            <p>Pas hier uw <a href="https://example.com">voorkeuren</a> aan voor de notificaties. </p>
+        <td class="td-mail td__padding-right td__padding-bottom td__padding-left" colspan="6">
+            <p>Pas hier uw <a href="{{ profile_notifications }}">voorkeuren</a> aan voor de notificaties. </p>
```

Comment:
⚠️ NOTE: Extra aandacht voor deze aanpassing, die technisch werkt als men dit ook werkelijk dusdanig in het HTML template toevoegt.